doc/install/install.rst

```diff
@@ -1,296 +1,147 @@
+    source deactivate
 
-The QIIME base install allows use of core QIIME functionality, including, but not limited to:
+If you decide later that you don’t want the environment or its packages
+anymore, deactivate the environment and then run this command:
 
- - `validate_mapping_file.py <../scripts/validate_mapping_file.html>`_
- - `split_libraries.py <../scripts/split_libraries.html>`_
- - `split_libraries_fastq.py <../scripts/split_libraries_fastq.html>`_
- - `pick_open_reference_otus.py <../scripts/pick_open_reference_otus.html>`_
- - `pick_de_novo_otus.py <../scripts/pick_de_novo_otus.html>`_
- - `pick_closed_reference_otus.py <../scripts/pick_closed_reference_otus.html>`_
- - `core_diversity_analyses.py <../scripts/core_diversity_analyses.html>`_
+::
 
-Prepare your build environment...
----------------------------------
+    conda remove --name qiime191 --all
 
-First, you need to have a build environment on your machine:
+These instructions have been tested on Mac OS X (10.8–10.11) and Linux
+(Ubuntu 14.04 and Mint 17.3 Rosa).
 
- - On OS X, this involves installing the `developer tools <http://developer.apple.com/technologies/xcode.html>`_.
- - On Debian-based Linux (e.g., Ubuntu), this involves installing the following packages::
+Installing QIIME dependencies in your Conda environment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-     sudo apt-get install build-essential python-dev python-pip
+You now have the base installation and access to much of QIIME's
+functionality. However, if you want to use additional dependencies, your
+Conda installation of QIIME will need to have access to the executable
+files for each program you'd like to invoke as a dependency.
+Commonly-used QIIME dependencies include UCLUST (now included in the
+base installation), RDP Classifier, BLAST, USEARCH, and VSEARCH.
 
-.. warning:: Depending on your system, additional packages may need to be installed that may be required by QIIME or its dependencies. Also, packages names will vary across operating systems and even within operating system distributions (e.g., different versions of Ubuntu). Since there are a wide range of systems that QIIME can be installed on, the QIIME development team cannot provide a comprehensive list of commands that are guaranteed to successfully install QIIME on your system.
+Linking to executables you already have
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-QIIME is written in Python 2, and requires Python version greater than Python 2.7.0 and less than Python 3.0.0. You can verify which version of Python you're running with::
+If you already have the appropriate version of these programs installed,
+you can ensure that they run the correct executable by making a symbolic
+link in the bin folder of your Conda QIIME environment. To allow your
```

Comment:
Won't these work if they're just in your ``$PATH``? It doesn't seem right to me to recommend messing with the conda environment-specific directories. What do the conda docs recommend for this?